code/__DEFINES/colors.dm

```diff
@@ -169,3 +169,13 @@
 #define LIGHT_COLOR_TUNGSTEN   "#FAE1AF"
 /// Barely visible cyan-ish hue, as the doctor prescribed. rgb(240, 250, 250)
 #define LIGHT_COLOR_HALOGEN    "#F0FAFA"
+
+//The GAGS greyscale_colors for each department's computer/machine circuits
+#define CIRCUIT_COLOR_GENERIC list("#1A7A13")
+#define CIRCUIT_COLOR_COMMAND list("#1B4594")
+#define CIRCUIT_COLOR_SECURITY list("#9A151E")
+#define CIRCUIT_COLOR_SCIENCE list("#BC4A9B")
+#define CIRCUIT_COLOR_SERVICE list("#92DCBA")
+#define CIRCUIT_COLOR_MEDICAL list("#00CCFF")
+#define CIRCUIT_COLOR_ENGINEERING list("#F8D700")
+#define CIRCUIT_COLOR_SUPPLY list("#C47749")
```

Comment:
I don't think these defines should be lists.

You should turn these into generic color defines (like the rest of this file), then just make a list everytime you call greyscale_colors.

In the way it is now, it's confusing--`greyscale_colors = CIRCUIT_COLOR_ENGINEERING` looks like `CIRCUIT_COLOR_*` is a normal color, not a list. You could change the variable names to match, as well, but I like the other approach better.